README.md

```diff
@@ -11,7 +11,7 @@ table of contents
     2. [installing mediamicroservices](https://github.com/mediamicroservices/mm#installing-mediamicroservices)
     3. [configuring mediamicroservices](https://github.com/mediamicroservices/mm#configuring-mediamicroservices)
         1. [variable explanations](https://github.com/mediamicroservices/mm#variable-explanations)
-3. [database configuration](#configuring-premisfixity-logging-database)
+3. [Database Instructions](#database-instructions)
```

Comment:
good point! I actually have been debating going through the doc and changing it to use capitalization across the board.